apps/portals/src/configurations/adknowledgeportal/synapseConfigs/target_enabling_resources.ts

```diff
@@ -39,7 +39,6 @@ export const targetEnablingResourcesCardConfiguration: CardConfiguration = {
 
 export const targetEnablingResourcesTableConfiguration: QueryWrapperPlotNavProps['tableConfiguration'] =
   {
-    title: 'Target Enabling Resources',
```

Comment:
Removes the title from the Synapse Table?

Reply:
Seems fine for this case (where the tab name indicates the title), but may be confusing in a Details page (where there may be multiple tables)

Reply:
I removed this `title` prop from `SynapseTableProps` because it was unused prior to my changes. We have to remove it here or else there's a type error. I think in most of these cases, the title is set in a different place.